cruise-control/src/yaml/responses/clusterModelStats.yaml

```diff
@@ -32,15 +32,15 @@ ClusterModelStatsValue:
     - AVG
     - MAX
     - MIN
-    - ST_DEV
+    - STD
```

Comment:
Is this change intentional? Shouldn't we have a test/sanity check that fails due to mismatch between `ST_DEV` and `STD`?

Reply:
This is before the [latest commit](https://github.com/linkedin/cruise-control/pull/1025/commits/fbfa8eebc29003e064dd24da6f4d32198f84903f) it is `ST_DEV` in both Java class and `yaml` file, and this commit make change in both place at the same time.